src/node/resolver.ts

```diff
@@ -155,9 +156,9 @@ export function createResolver(
     }
   }
 
-  resolvers.forEach((r) => {
-    if (r.alias && typeof r.alias === 'object') {
-      resolveAlias(r.alias)
+  resolvers.forEach(({ alias }) => {
```

Comment:
Shouldn't we type this as `Resolver` instead of just relying on implicitness?

Reply:
No need to explicit type what can be inferred. You can hover over it to see the type anyway.